bin/main.rb

```diff
@@ -0,0 +1,33 @@
+puts 'welcome to Tic Tac Toe'
+puts 'Enter your Name (First Player): '
+player_one = gets.chomp
+# save the player_one name in this var
+puts "#{player_one} your symbol is X "
+
+puts 'Enter your Name (Second Player): '
+player_two = gets.chomp
+# save the player_one name in this var
+puts "#{player_two} your symbol is O "
+
+9.times do
+  puts '-------------'
+  puts '| a | b | c |'
+  puts '| d | e | f |'
+  puts '| g | h | i |'
+  puts '-------------'
+end
```

Comment:
- [ ] A loop needs to be terminated at some point (after a winning move or draw to be more specific). It can be done with variable:
Use this implementation for inspiration:
```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

I, therefore, would advise us not to use the `out-of-nowhere` numbers to terminate a loop as you have done here. Kindly avoid using it.